doc/source/user/how-to-how-to.rst

```diff
@@ -0,0 +1,114 @@
+How to write a NumPy how-to
+##############################################################################
+
+How-tos get straight to the point -- they
+
+  - answer a focused question, or
+  - narrow a broad question into focused questions that the user can
+    choose among.
+
+******************************************************************************
+A stranger has asked for directions...
+******************************************************************************
+
+**"I need to fill my tank."**
+
+******************************************************************************
+Give a brief but explicit answer
+******************************************************************************
+
+  - `"Three miles, take a right at Hayseed Road, it's on your left."`
+
+Add helpful details for newcomers ("Hayseed Road", even though it's the only
+turnoff at three miles). But not irrelevant ones:
+
+  - Don't also give directions from Route 7.
+  - Don't explain why the town has only one gas station.
+
+If there's related background (tutorial, explanation, reference, alternative
+approach), bring it to the user's attention with a link ("Directions from Route 7,"
+"Why so few gas stations?").
+
+
+******************************************************************************
+Delegate
+******************************************************************************
+
+  - `"Three miles, take a right at Hayseed Road, follow the signs."`
+
+If the information is already documented and succinct enough for a how-to,
+just link to it, possibly after an introduction ("Three miles, take a right").
+
+******************************************************************************
+If the question is broad, narrow and redirect it
+******************************************************************************
+
+ **"I want to see the sights"**
+
+The `See the sights` how-to should link to a set of narrower how-tos:
+
+- Find historic buildings
+- Find scenic lookouts
+- Find the town center
+
+and these might in turn link to still narrower how-tos -- so the town center
+page might link to
+
+   - Find the court house
+   - Find city hall
+
+By organizing how-tos this way, you not only display the options for people
+who need to narrow their question, you also have provided answers for users
+who start with narrower questions ("I want to see historic buildings," "Which
+way to city hall?").
+
+******************************************************************************
+If there are many steps, break them up
+******************************************************************************
+
+If a how-to has many steps:
+
+  - Consider breaking a step out into an individual how-to and linking to it.
+  - Include subheadings. They help readers grasp what's coming and return
+    where they left off.
+
+******************************************************************************
+Why write how-tos when there's Stack Overflow, Reddit, Gitter...?

```

Comment:
Maybe condition their existence on the premise that we have an authoritative answer, something like "we will only write how-tos if we have authoritative answers"

Reply:
Could you clarify? I'm not sure I understand what's wrong or what the proposed rewording is.